frontend/src/app/gaming/dropin/dropin.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-dropin',
+  templateUrl: './dropin.component.html',
+  styleUrls: ['./dropin.component.scss']
+})
+export class DropinComponent implements OnInit {
+  available = false;
```

Comment:
Is this meant to be an Input()?

Reply:
no! it's a thing that's going to figure out in ng oninit whether or not it should be displaying anything at all. i really haven't figured out how to do layout effectively for this page